### PR TITLE
Fix pytorch lightning version to 0.9.0

### PR DESCRIPTION
**Pull request by titu1994**
Squad fix by @yzhang123 
- removed 1 of many squad tests, since all components are covered by the rest of the tests.
- patched the cause of the test failure - basically needs to be run 1 full epoch to properly calculate validation accuracy

Signed-off-by: smajumdar <smajumdar@nvidia.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -247,13 +247,11 @@ pipeline {
             model.train_ds.batch_size=8 \
             model.validation_ds.batch_size=8 \
             trainer.max_epochs=1 \
-            +trainer.max_steps=1 \
```
Comment (blisc) [inline Jenkinsfile:250]: Why did we remove max_steps=1?

Comment (yzhang123) [reply Jenkinsfile:250]: basically it calculates validation accuracy on the full validation dataset, this is SQuAD specific

Comment (blisc) [reply Jenkinsfile:250]: max_steps is only for training steps.
num_sanity_val_steps is for validation steps, which in lightning 0.9 is now -1 and does a full run.
I'm pretty sure you can add max_steps=1 back in

Comment (titu1994) [reply Jenkinsfile:250]: @yzhang123 Could you open a PR after this to see if we can add back max_steps=1?

Comment (yzhang123) [reply Jenkinsfile:250]: @blisc oh yes, the ci had also num_sanity_val_steps limited, so probably thats why it broke
